src/System.Net.Primitives/tests/PalTests/SocketAddressPalTest.cs

```diff
@@ -61,9 +61,19 @@ public void Port_Get_Set_IPv6(ushort port)
         }
 
         [Fact]
-        public void Port_Get_Set_Throws()
+        public void Port_Get_Set_Throws_IPv4()
```

Comment:
It's ok to have them separate (especially if we expect there might be further divergence between them in the future), but did you consider using a theory instead? e.g.
```C#
[Theory]
[InlineData(AddressFamily.InterNetwork)]
[InlineData(AddressFamily.InterNetworkV6)]
public void Port_Get_Set_Throws(AddressFamily family)
{
    var buffer = new byte[2];
    SocketAddressPal.SetAddressFamily(buffer, family);
    Assert.ThrowsAny<Exception>(() => SocketAddressPal.SetPort(buffer, 0));
    Assert.ThrowsAny<Exception>(() => SocketAddressPal.GetPort(buffer));
}
```